notebooks/pudl-catalog.ipynb

```diff
@@ -6,7 +6,27 @@
     "tags": []
```

Comment:
<p>I mention this below, but when going through this notebook it's not that clear when you access the data for the first time and when the cache is created. I guess it wasn't clear to be that accessing the metadata associated with the database requires accessing the data.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl-catalog/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>